src/index.js

```diff
@@ -56,7 +56,7 @@ document.addEventListener("DOMContentLoaded", () => {
 
         // Message container
         const message = document.createElement("article");
-        message.classList.add("message", "is-primary");
+        message.classList.add("message", done ? "is-secondary" : "is-primary");
```

Comment:
There is no "is-secondary" color class in bulma. Perhaps you meant to say "is-dark"?

Reply:
Fixed in #36699a6. Sorry for the mistake. it looke dark in the dev server 😃